File: core/src/com/mygdx/game/ship/CleanCorridor.java
package com.mygdx.game.ship;

/**
 * Created by Kristen on 5/9/2016.
 */
public class CleanCorridor extends Tile {
    public CleanCorridor(){
        this(null);
    }

    public CleanCorridor(Ship ship) {
        super(ship);
        CleanCorridorRenderObject ro = new CleanCorridorRenderObject(this);
        type = TileType.Walkable;
    }
}

File: web/src/ship/CleanCorridor.ts
import Phaser from 'phaser';
import { Tile } from './Tile';
import { TileType } from './TileType';

/** Same as Corridor (floor-only deck). */
export class CleanCorridor extends Tile {
  constructor(scene: Phaser.Scene) {
    super(scene);
    this.type = TileType.Walkable;

    const floor = scene.make.image({ x: 0, y: 0, key: 'floor' }).setOrigin(0, 0);
    this.add(floor);
  }
}
